.github/workflows/github_action_test_suite.yml

```diff
@@ -66,27 +68,19 @@ jobs:
       - name: Run test suite
         if: ${{ ! matrix.coverage-test }}
         working-directory: testing_mygame
+        env:
+          UNIT_TEST_SETTINGS: "--settings=settings --keepdb --parallel 4 --timing evennia"
         run: |
-          evennia test \
-            --settings=settings \
-            --keepdb \
-            --parallel 4 \
-            --timing \
-            evennia
+          evennia test ${{ env.UNIT_TEST_SETTINGS }}
 
       # OBS - it's important to not run the coverage tests with --parallel, it messes up the coverage
       # calculation!
       - name: Run test suite with coverage
         if: ${{ matrix.coverage-test }}
         working-directory: testing_mygame
         run: |
-          coverage run \
-            --source=evennia \
-            --omit=*/migrations/*,*/urls.py,*/test*.py,*.sh,*.txt,*.md,*.pyc,*.service \
-            ../bin/unix/evennia test \
-              --settings=settings \
-              --timing \
-              evennia
+          coverage run --rcfile=../pyproject.toml
+          coverage combine
```

Comment:
While I know this the details are covered in  `[tool.coverage.run]` in `pyproject.toml`, breakiing it out like this makes this file harder to read imo; while the `pyproject.toml` trick is nice, it seems like it would be easier to understand this file if this was also using the `UNIT_TEST_SETTINGS` envvar. 

If keeping it in the pyproject.toml file, we need at least some comment here to explain this.

Reply:
TL;DR - coverage testing has its own environment variable for the settings for now

I met you halfway here, I think.  I took out `--parallel` from the coverage test for now, because I can't resolve the issue with the `module-not-measured` warning probably until after I figure out how to get WSL working properly on my rig.  I left the general coverage settings in the TOML (the source and omissions) though, which is the "halfway" part of my statement.